packages/devtools/lib/src/logging/logging.dart

```diff
@@ -565,7 +565,9 @@ class LogWhenColumn extends Column<LogData> {
 
   @override
   String render(dynamic value) {
-    return timeFormat.format(DateTime.fromMillisecondsSinceEpoch(value));
+    return value == null
```

Comment:
Makes sense to guard against this, but I'm curious why we have logs without timestamps?

Reply:
I think this is coming from the webdev vm service debugger proxy.

@jakemac53 @grouma

Reply:
I wouldn't be surprised if we are missing something here as we don't currently have good validation around "required" fields. Can take a look.